diff --git a/dist/cli.cjs.dev.js b/dist/cli.cjs.dev.js
index 73ab02a861b1f5a8e1bf10984340a0a6b1518b15..2309d78fb1ff07428bc76136d9eb4f4d8d6571cc 100644
--- a/dist/cli.cjs.dev.js
+++ b/dist/cli.cjs.dev.js
@@ -279,6 +279,9 @@ async function confirmMajorRelease(pkgJSON) {
 }
 
 async function getPackagesToRelease(changedPackages, allPackages) {
+  const isPrivate = (name) => name === 'astro-scripts' || name.startsWith('@example/') || name.startsWith('@test/') || name.startsWith('@e2e/');
+  changedPackages = changedPackages.filter(name => !isPrivate(name));
+  allPackages = allPackages.filter(({ packageJson: { name }}) => !isPrivate(name));
   function askInitialReleaseQuestion(defaultChoiceList) {
     return askCheckboxPlus( // TODO: Make this wording better
     // TODO: take objects and be fancy with matching