diff --git a/dist/assemble-release-plan.cjs.dev.js b/dist/assemble-release-plan.cjs.dev.js
index f05ea74cd2a76884ed8a754e821775003af63764..de0b7fe8d53ffad29cae2de145042b9b8376625c 100644
--- a/dist/assemble-release-plan.cjs.dev.js
+++ b/dist/assemble-release-plan.cjs.dev.js
@@ -65,6 +65,9 @@ function incrementVersion(release, preInfo) {
   }
 
   let version = semver.inc(release.oldVersion, release.type);
+  if (release.name === 'astro') {
+    version = semver.inc(release.oldVersion, 'prerelease');
+  }
 
   if (preInfo !== undefined && preInfo.state.mode !== "exit") {
     let preVersion = preInfo.preVersions.get(release.name);